### ISSUE TITLE
kitti splits

### ISSUE THREAD

**Issue by sean-metzger** [google-research/task_adaptation#9]
it seems like the splits on kitti don't match what's in the VTAB paper. 

the kitti splits obtained in kitti.py seem to be the default splits from tfdatasets, which results in a training set with size 6,347, a val set with size 423, and a test set of 711 samples. However, in the VTAB paper, the splits reported are using 80% of the training set for train, which is the size returned in kitti_test.py, but not when you actually make the data loader. 
Unlike in resisc.py, for example, in kitti.py the train and val splits are not defined, leading me to believe the train and test splits from tfdatasets were what was actually used. Was this the case?

**Comment by neilhoulsby** [comment]
Yes, you are right the splits from tfds we're used for the "full dataset" splits. There was a bug in the counting of the examples, which I believe would slightly effect the number of steps run if the schedule was defined in terms of epochs. This should be fixed in the latest update.
